CheckYourEligibility.API/Boundary/Requests/ApplicationBulkImportRequest.cs

```diff
@@ -0,0 +1,68 @@
+// Ignore Spelling: Fsm
+
+using CheckYourEligibility.API.Domain.Enums;
+using Microsoft.AspNetCore.Http;
+
+namespace CheckYourEligibility.API.Boundary.Requests;
+
+/// <summary>
+/// Request for bulk importing applications from a CSV or JSON file
+/// </summary>
+public class ApplicationBulkImportRequest
+{
+    /// <summary>
+    /// The CSV or JSON file containing application data
+    /// </summary>
+    public required IFormFile File { get; set; }
```

Comment:
I think we need to consider this as a JSON endpoint, not somewhere we upload files.